test/instaskip/impl/eskip_map_process_facts.clj

```diff
@@ -0,0 +1,102 @@
+                                                             :type  "string"}]}]) =>
+             {:uses-common-filters false
+              :filters             [{:name "filter1" :args []}
+                                    {:name "filter2" :args [{:value "value"
+                                                             :type  "string"}]}]}))
+
+(facts "split-hosts"
+       (fact "splits a regex string into more hosts"
+             (split-hosts "/^(host1[.]com|host2[.]com|host3[.]com)$/") =>
+             ["host1.com" "host2.com" "host3.com"]))
+
+(facts "eskip-map->route-with-path"
+       (fact "transforms from eskip map to innkeeper map"
+             (eskip-map->route-with-path
+               "theTeam"
+               {:name       "theRoute"
+                :predicates [{:name "Host"
+                              :args [{:value "/^(host1[.]com|host2[.]com)$/"
+                                      :type  "regex"}]}
+                             {:name "Path" :args [{:value "/path/example"
+                                                   :type  "string"}]}
+                             {:name "Method" :args [{:value "GET"
+                                                     :type  "string"}]}]
+                :filters    [{:name "filter1" :args []}
+                             {:name "fashionStore" :args []}
+                             {:name "filter2" :args [{:value "value"
+                                                      :type  "string"}]}]
+                :endpoint   ""}) =>
+
+             {:route
+                    {:name                "theRoute",
+                     :predicates          [{:name "Method"
+                                            :args [{:type "string" :value "GET"}]}]
+                     :filters             [{:name "filter1"
+                                            :args []}
+                                           {:name "filter2"
+                                            :args [{:type "string" :value "value"}]}]
+                     :endpoint            ""
+                     :uses-common-filters true}
+              :path {:uri           "/path/example"
+                     :hosts         ["host1.com" "host2.com"]
+                     :owned-by-team "theTeam"}})
+
+       (fact "transforms from eskip to innkeeper a real route"
+             (eskip-map->route-with-path
+               "theTeam"
+               {:name       "aladdin_genieWishlistItemsApi",
+                :predicates [{:name "Path", :args [{:value "/api/wishlist", :type "string"}]}
+                             {:name "Host",
+                              :args [{:value "/^(m-it[.]release[.]zalando[.]net|m-pl[.]release[.]zalando[.]net)$/",
+                                      :type  "regex"}]}],
+                :filters    [{:name "fashionStore", :args []}],
+                :endpoint   "https://genie.aladdin-staging.zalan.do"}) =>
```

Comment:
not sure, we want to "leak" internal routes to the public? can we use anonymised test cases?